lib/validations.js

```diff
@@ -383,6 +384,18 @@ function validateUniqueness(attr, conf, err, options, done) {
   }.bind(this));
 }
 
+/*!
+ * Date validator
+ */
+function validateDate(attr, conf, err) {
```

Comment:
the `validators` are defined in the block around line 397.  There are `validatePresence`, `validateAbsence` and also the newly added `validateDate`.  If you go to those `validate*` functions, they all have `conf`.  This is needed when those functions are being called.
In fact, without `conf`, tests fail because it expects the 2nd parameter in validate* function to be a function.